salt/runners/net.py

```diff
@@ -236,7 +236,7 @@ def _find_interfaces_mac(ip):  # pylint: disable=invalid-name
         for interface, interface_ipaddrs in six.iteritems(
             device_ipaddrs.get("out", {})
         ):
-            ip_addresses = interface_ipaddrs.get("ipv4", {}).keys()
+            ip_addresses = list(interface_ipaddrs.get("ipv4", {}))
             ip_addresses.extend(interface_ipaddrs.get("ipv6", {}).keys())
```

Comment:
The call to `keys` is redundant here. `list.exted()` works good with dicts using the only keys.

Reply:
Ah this is from the other PR I included then removed.